frontend/components/common/Layout/Footer.tsx

```diff
@@ -286,32 +291,39 @@ function NewsletterForm({
                <FooterNewsletterEmailLabel>
                   Enter your email
                </FooterNewsletterEmailLabel>
-               <FooterNewsletterEmailInput
-                  ref={inputRef}
-                  disabled={subscription.status !== SubscriptionStatus.Idle}
-                  placeholder={emailPlaceholder}
-               />
+               <HStack>
+                  <FooterNewsletterEmailInput
+                     ref={inputRef}
+                     disabled={subscription.status !== SubscriptionStatus.Idle}
+                     placeholder={emailPlaceholder}
+                     hidden={isSubscribed}
+                  />
+                  {isSubscribed && (
+                     <Text
+                        align="center"
+                        position={'absolute'}
+                        top="2"
+                        left="0"
+                        right="0"
+                     >
+                        <Icon as={RiCheckFill} boxSize="5" mb="-5px" mr="6px" />
+                        Subscribed!
+                     </Text>
+                  )}
+                  <Button
+                     type="submit"
+                     isLoading={
+                        subscription.status === SubscriptionStatus.Subscribing
+                     }
+                     disabled={subscription.status !== SubscriptionStatus.Idle}
+                     colorScheme="brand"
+                     visibility={isSubscribed ? 'hidden' : undefined}
+                  >
+                     Subscribe
```

Comment:
Should this be subscribeLabel? I wonder if this needs to be a variable at all, but since we have it, might as well use it.

Reply:
Sure! 5d0825b4c03dff078da8a1352c2dfa25f25e7054